### PR TITLE
🌙 nightly logo

### PR DESCRIPTION
**Pull request by alex-wilmer**

### CODE REVIEW

**Comment by alex-wilmer** [comment]
image

File: src/packages/@ncigdc/components/Header.js [hunk 1]
```diff
@@ -25,6 +21,10 @@ import SessionExpiredModal from '@ncigdc/components/Modals/SessionExpiredModal';
 import Banner from '@ncigdc/uikit/Banner';
 import { withTheme } from '@ncigdc/theme';
 
+const nciGdcLogo = process.env.REACT_APP_ENV === 'nightly'
```
Comment (shanewilson) [inline src/packages/@ncigdc/components/Header.js]: should probably use the constants file for everything like this

**Comment by rjackson90** [comment]
If anyone's interested, here's the corresponding PR in tungsten: https://github.com/NCI-GDC/tungsten/pull/595

**Comment by alex-wilmer** [comment]
thanks @rjackson90. @shanewilson any objections?

**Pull request closed without merging**